src/Sylius/Behat/Context/Setup/CurrencyContext.php

```diff
@@ -93,11 +93,17 @@ public function theStoreHasCurrency($currencyCode)
 
     /**
      * @Given the store has currency :currencyCode, :secondCurrencyCode
+     * @Given the store has currency :currencyCode and :secondCurrencyCode
+     * @Given the store has currency :currencyCode, :secondCurrencyCode and :thirdCurrencyCode
      */
-    public function theStoreHasCurrencyAnd($currencyCode, $secondCurrencyCode)
+    public function theStoreHasCurrencyAnd($currencyCode, $secondCurrencyCode, $thirdCurrencyCode = null)
```

Comment:
In case of `storeShipsTo` there's no transformer in the way. Here, we pass currency name in the scenario, but get currency code in return. Only way to actually use variadics in here would be to duplicate the name => code conversion from the transformer, your call.